simpleParser.go

```diff
@@ -387,10 +405,28 @@ func (ctx *expressionParserContext) mergeAndRestoreSubContexts(currentSubCtx *pa
 	return nil
 }
 
+func (ctx *expressionParserContext) enableShortCircuitEvalIfPossible() {
+	// Enable short circuit if expression is consisted only of && or only of ||
+	var operator string
+	for i := 0; i < len(ctx.tokens); i++ {
+		if ctx.tokens[i] == TokenOperatorOr || ctx.tokens[i] == TokenOperatorAnd {
```

Comment:
can there be other operators other than "or" and "and"? if so should short circuit be disabled?

Reply:
That's a good point. Let's enable it if there's at least one or more matching || or &&.
Should be safer going forward.